Week2/Homework/assets/question-six.js

```diff
@@ -1,10 +1,17 @@
 let nlCities = [];
-console.log('the value of the array is 0');
+console.log('the value of the array is undefined');
```

Comment:
Ah this one is not correct yet, the value is not `undefined` :) So check it again